services/support.md

```diff
@@ -1,4 +1,19 @@
 | Action | Description | Resource | Condition |
 | --- | --- | --- | --- |
-| [support:*](http://docs.aws.amazon.com/awssupport/latest/user/getting-started.html#accessing-support) | Support does not let you allow or deny access to individual actions; therefore, the Action element of a policy is always set to support:*. Similarly, Support does not provide resource-level access, so the Resource element is always set to *. An IAM user with Support permissions has access to all Support operations and resources. | * | - |
-
+| [support:AddAttachmentsToSet](https://docs.aws.amazon.com/awssupport/latest/APIReference/API_AddAttachmentsToSet.html) | Adds one or more attachments to an AWS Support case. | * | - |
+| [support:AddCommunicationToCase](https://docs.aws.amazon.com/awssupport/latest/APIReference/API_AddCommunicationToCase.html) | Adds a customer communication to an AWS Support case. | * | - |
+| [support:CreateCase](https://docs.aws.amazon.com/awssupport/latest/APIReference/API_CreateCase.html) | Creates a new AWS Support case. | * | - |
+| [support:DescribeAttachment](https://docs.aws.amazon.com/awssupport/latest/APIReference/API_DescribeAttachment.html) | Returns the description for an attachment. | - |
```

Comment:
Should end with `| * | - |`?

Reply:
Yup, missed that by mistake. Added a new commit with that fixed.